src/renderer/navigation/events.ts

```diff
@@ -39,7 +39,7 @@ function bindClusterManagerRouteEvents() {
 
 // Handle cluster-view renderer process events within iframes
 function bindClusterFrameRouteEvents() {
-  subscribeToBroadcast("renderer:navigate-cluster-view", (event, url: string) => {
+  subscribeToBroadcast("renderer:navigate-in-cluster", (event, url: string) => {
```

Comment:
Would you mind extracting this to a constant instead of being a magic string?

Reply:
Agreed if we'll use constants consistently.